### ISSUE TITLE
Unable to override get_audit_schema macro 

### ISSUE THREAD

**Issue by bookerserge** [fishtown-analytics/dbt-event-logging#28]
### Describe the bug

Unable to override get_audit_schema macro with custom logic

### Steps To Reproduce

1. add  logging to  packages.yml
```     
    packages:
  - package: fishtown-analytics/logging
    version: 0.2.3

```
2. add following hooks to dbt_project.yml
```
      pre-hook: "{{ logging.log_model_start_event() }}"
     post-hook: "{{ logging.log_model_end_event() }}"
```
2. create audit_schema_name.sql macro in macro directory
```
    {% macro get_audit_schema() %}

       {{ return('analytics_meta') }}

    {% endmacro %}
```
3. run dbt deps

4. dbt run

### Expected behavior

I would expect logging to be created in analytics_meta schema

### Actual behavior

logging is created in [target.schema]_meta schema

### System information
**How did you add this package to your project:**
```
    packages:
  - package: fishtown-analytics/logging
    version: 0.2.3
```

**Which database are you using dbt with?**
- [ ] Postgres
- [ ] Redshift
- [ ] BigQuery
- [X] Snowflake
- [ ] Other (specify: ____________)

**The output of `dbt --version`:**
```
installed version: 0.17.0
   latest version: 0.17.0

Up to date!

Plugins:
  - bigquery: 0.17.0
  - postgres: 0.17.0
  - redshift: 0.17.0
  - snowflake: 0.17.0
```

### Additional context

I'm not sure if this issue could be caused by my  generate_schema_name macro override but here the sql just in case

{% macro generate_schema_name(custom_schema_name, node) -%}

  {% set default_schema = target.schema  -%}
  
  {% if custom_schema_name is  none -%}
        {{ default_schema }}
  {%- else -%}  
        {{ custom_schema_name | trim }}
  {% endif %}

{% endmacro %}

**Comment by clrcrl** [comment]
Just want to clarify something:
> I would expect logging to be created in analytics_meta schema
> logging is created in [target.schema]_meta schema

Do you mean:
a. The logging models, e.g. `stg_dbt_audit_log` are created in `<target.schema>_meta.stg_dbt_audit_log`?:
From the command line:
```
11:31:22 | 1 of 5 START view model dev_ccarroll_meta.stg_dbt_audit_log.......... [RUN]
```
or
b. The log events are inserted into `<target.schema>_meta.dbt_audit_log `
From `logs/dbt.log`:

```
insert into analytics_meta.dbt_audit_log (
        event_name,
        event_timestamp,
        event_schema,
        event_model,
        event_user,
        event_target,
        event_is_full_refresh,
        invocation_id
    )
...
```

The reason I ask this is because the `get_audit_schema` macro only changes the schema name of the target table for the events (i.e. the table that dbt `inserts into`).

If you want to change the schema of the logging models, you can do this in your `dbt_project.yml` like so:
```
models:
  logging:
    schema: my_schema
```

**Comment by bookerserge** [comment]
Hi Claire,

I meant that the logging events are inserted into <target.schema>_meta.dbt_audit_log. 

from my log...
```

insert into <target.database>.<target.schema>_meta.dbt_audit_log (
        event_name,
        event_timestamp,
        event_schema,
        event_model,
        event_user,
        event_target,
        event_is_full_refresh,
        invocation_id
    )

    values (
        'run completed',
        
    convert_timezone('UTC', 
    current_timestamp::
    timestamp_ntz

)::
    timestamp_ntz

,
        '',
        '',
        '****************@***********',
        'dev',
        FALSE,
        '*******************************'
    );

```

**Comment by clrcrl** [comment]
Thank you! Super useful info!

**Comment by bookerserge** [comment]
This seems resolved in 0.3.0. I'm going to close the issue. Thanks again!
